internal/scorer/config.go

```diff
@@ -121,6 +123,13 @@ func LoadConfig(r io.Reader) (*Config, error) {
 	if err := yaml.Unmarshal(data, c); err != nil {
 		return nil, err
 	}
+
+	for _, i := range c.Inputs {
+		if i.Weight <= 0 {
+			return nil, wam.ErrWeightLessThanOrEqualToZero
+		}
```

Comment:
See my comment re:this error check.

Additionally, the error `ErrWeightLessThanOrEqualToZero` should not be pulled from `wam`.

Reply:
👍